contrib/limine/template.py

```diff
@@ -0,0 +1,25 @@
+pkgname = "limine"
+pkgver = "4.20230422.0"
+pkgrel = 0
+archs = ["x86_64", "aarch64", "!ppc*", "!riscv*"]
+build_style = "gnu_configure"
+configure_args = [
+    "--prefix=/usr",
```

Comment:
no need to specify this (`gnu_configure` does it by default)

Reply:
fixed